README.md

```diff
@@ -1,59 +1,72 @@
+repositories. These also correspond to some of the default integrations
+available to all Daml Hub users via the _Browse Integrations_ tab in the
+[console](https://hub.daml.com/docs/quickstart)'s ledger view.
+
+* [Core Pack](https://github.com/digital-asset/daml-dit-integration-core)
+* [Coindesk](https://github.com/digital-asset/daml-dit-integration-coindesk)
+* [Slack](https://github.com/digital-asset/daml-dit-integration-slack)
+* [Exberry](https://github.com/digital-asset/daml-dit-integration-exberry)
+
+
+## Integration Packaging and Deployment
+
+Integrations are packaged in [DIT files](https://github.com/digital-asset/daml-dit-api),
+built using the [`ddit` build tool](https://github.com/digital-asset/daml-dit-ddit),
+and can be deployed into Daml Hub using the same upload mechanism as other
+artifacts (Daml Models, bots, etc.). Daml Hub also has an 'arcade' facility
+that uses a public [GitHub repository](https://github.com/digital-asset/daml-dit-arcade-index)
+to maintain list of integrations and sample apps that can be deployed through
+a single click on the [console user interface](https://hub.daml.com/docs/quickstart).
+
+Logically speaking, DABL integrations package their Python
+implementation code alongside metadata describing available
+integration types and any other resources required to make the
+integration operate. Integrations usually require Daml models to
+represent their data model, and often require external Python
+dependencies specified through a `requirements.txt` file. Both of
+these are examples of additional resourced that might be bundled into
+a DIT file.  Note that while `daml-dit-if` and `dazl` are both Python
+dependencies required by integrations, they are exceptions to this
+rule.  Daml Hub provides both of these by default, so they should not
+be listed in `requirements.txt` and should not be included in the DIT
+file.
 
 ## The Integration Framework API
 
-The integration framework API has two parts - a Python entry point
-that all integrations must provide and an additional section within
-`dabl-meta.yaml` that describes the properties of a given
-integration. The metadata section includes the name of the entry point
-function for the integration, some descriptive text, and a list of
-all of the configuration arguments that the integration accepts:
+The integration framework API has two parts. First, there is an
+additional `integration_types` section within `dabl-meta.yaml` that
+lists and describes the integration types defined within the DIT file.
+This metadata section includes the name of the entry point function
+for the integration type, some descriptive text, and a list of all of
+the configuration arguments that the integration accepts:
 
-The ledger event log integration is defined like this:
+The [ledger event log integration](https://github.com/digital-asset/daml-dit-integration-core/blob/master/src/core_int/integration_ledger_event_log.py) is defined like this:
```

Comment:
This might be better as a permalink (instead of pointing to `master`).

Reply:
I see your point, but I wrote it this way, preferring to link to the current latest.